### ISSUE TITLE
🐛 "An input component must either have a 'aria-label' attribute..." error when a <label> is present. 

### ISSUE THREAD

**Issue by tjosepo** [gsoft-inc/sg-orbit#1229]
## Describe the bug

Input elements log an error about accessibility when `aria-label`, `aria-labelledby` and `placeholder` are missing.

There is no check for `<label>` elements, which makes some accessible inputs still log an error.

## Steps to reproduce

Create an element with this return:

```jsx
<>
    <Label htmlFor="test">Label</Label>
    <TextInput id="test" />
</>
```

## Expected results

There should be no errors. The text input has an accessible label.
